installer/install-rust-analyzer.sh

```diff
@@ -6,14 +6,15 @@ os=$(uname -s | tr "[:upper:]" "[:lower:]")
 
 case $os in
 linux)
-  platform="linux"
+  platform="x86_64-unknown-linux-gnu"
```

Comment:
`aarch64` architecture also exists. Maybe you can also declare the following variable above.

```sh
arch=$(uname -p)
```